src/python/director/perception.py

```diff
@@ -518,6 +518,22 @@ def showMap(self, viewId, mapId):
     def getSceneHeightData(self):
         return self.getDepthMapData(lcmdrc.data_request_t.HEIGHT_MAP_SCENE)
 
+    def getOctreeWorkspaceData(self):
+        return self.getPointCloudData(lcmdrc.data_request_t.OCTREE_WORKSPACE)
+
+    def getPointCloudData(self, viewId):
```

Comment:
I'd prefer this method return the polyData object.  The caller can extract the points if they want.